src/main/java/com/spotinst/sdkjava/model/api/oceanCD/ApiRolloutSpecStrategy.java

```diff
@@ -0,0 +1,62 @@
+package com.spotinst.sdkjava.model.api.oceanCD;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+import com.spotinst.sdkjava.model.bl.oceanCD.BackgroundVerificationArgs;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiRolloutSpecStrategy implements IPartialUpdateEntity {
+    @JsonIgnore
+    private Set<String>                               isSet;
+    private String                                    name;
+    private List<ApiBackgroundVerificationArgs>          args;
```

Comment:
Should it be ApiRolloutSpecStrategyArgs

Reply:
I am re-using the same class used in Strategy